src/CodeGenerator/src/Generator/InputGenerator.php

```diff
@@ -561,7 +561,11 @@ private function stringify(string $variable, Member $member, string $part): stri
         $shape = $member->getShape();
         switch ($shape->getType()) {
             case 'timestamp':
-                $format = strtoupper($shape->get('timestampFormat') ?? ('header' === $part ? 'rfc822' : 'iso8601'));
+                $format = strtoupper($shape->get('timestampFormat') ?? ('header' === $part ? 'rfc822' : 'atom'));
```

Comment:
the fallback here should still be `iso8601`, to correspond to values used in `timestampFormat` IMO. The rewriting will be done below anyway

Reply:
That's a good point, I'll change that quickly.